### ISSUE TITLE
interact_sky cannot obtain nearby TICs when tpf is from tpf.cutout() call

### ISSUE THREAD

**Issue by orionlee** [lightkurve/lightkurve#1089]
#### Problem description
`interact_sky()` cannot obtain nearby TICs, if the tpf in question is obtained from a tpf.cutout() call.

#### Example

```python
import lightkurve as lk

# tpf is some TESS TargetPixelFile
tpf_cutout = tpf.cutout(center=(6, 8), size=(6, 3))
tpf_cutout.interact_sky();
```

The interact_sky plot is generated, with a warning indicating it cannot get nearby TICs:
```
lightkurve\interact.py:475: LightkurveWarning: interact_sky() - cannot obtain nearby TICs. Skip it. The error: 'GAIA DR2'
  LightkurveWarning,
```

#### Environment

-  platform (e.g. Linux, OSX, Windows): Windows
-  lightkurve version (e.g. 1.0b6): 2.0.10
-  installation method (e.g. pip, conda, source): conda

**Comment by orionlee** [comment]
Root cause:
- Nearby TIC report relies on `TICID` header to get the TIC. 
- However, for the tpf obtained from `tpf.cutout()` call, the `TICID` is in the form of `12345678_CUTOUT`, the existing codes cannot handle it.

**Comment by orionlee** [comment]
@barensten this issue can also be closed with PR #1088 merged - github's auto close didn't catch it.
